src/pages/Home/Home.module.css

```diff
@@ -0,0 +1,70 @@
+.homeBackground {
+    height: 100vh;
+    width: 100%;
+    background-image: url('../../assets/images/background.svg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.logoImage {
+    width: 311px;
+    height: 139px;
+}
+
+.highlights {
+    padding: 20px 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.highlights img {
```

Comment:
I generally find that CSS is easier to follow when each component has it's own CSS it's concerned with. So here the `.highlights img` and `.highlightsWrapper` could be in their own module.

The size of this CSS module is fine however when the modules grow it can be hard to maintain.

Reply:
Done &#9989;